app/controllers/control_panel/stories_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ControlPanel::StoriesController < ControlPanel::ControlPanelController
+
+  def index
+    board = Board.find(params[:board_id])
```

Comment:
create method `find_board`
```
private 

def find_board
  Board.find(params[:board_id])
end
```
and then you can use it: board = find_board
You can also read about before_action and use it but be careful, many people consider that is not the best practise ( find out why )